packages/components/src/components/tick-picker/tick-picker.scss

```diff
@@ -0,0 +1,51 @@
+.dc-tick-picker {
+    width: 100%;
+    max-width: 300px;
+    height: 328px;
+    background-color: inherit;
+    opacity: 0.64;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+
+    div {
+        display: flex;
+    }
+    button {
+        background-color: var(--general-main-1);
+        width: 100%;
+        max-width: 112px;
+
+        span {
+            color: var(--text-prominent);
+        }
+    }
+    &__calculation {
+        margin-bottom: 28px;
+        justify-content: space-evenly;
+        align-items: center;
+
+        .tick-holder {
```

Comment:
You can move this up one step and use `&__tick-holder` as well.

Reply:
it is in the `calculation` class, I think it makes more sense to stay there.